src/QueryBuilder/CommonFilterQueryBuilder.php

```diff
@@ -184,6 +185,49 @@ protected function processWhereFilters(
         }
     }
 
+    /**
+     * @param RepeatedField|WhereRefTableFilter[] $filters
+     */
+    protected function processWhereRefTableFilters(
+        QueryBuilder $query,
+        RepeatedField $filters,
+        string $tableName,
+    ): void {
+        foreach ($filters as $filter) {
+            if (!array_key_exists($filter->getOperator(), self::OPERATOR_MULTI_VALUE)) {
+                throw new QueryBuilderException(
+                    'Only IN or NOT INT operator is allowed.',
+                );
+            }
+
+            $column = sprintf(
+                '%s.%s',
+                BigqueryQuote::quoteSingleIdentifier($tableName),
+                BigqueryQuote::quoteSingleIdentifier($filter->getColumn()),
+            );
+
+            $subTable = $filter->getRefTable();
+            $subColumn = sprintf(
+                '%s.%s',
+                BigqueryQuote::quoteSingleIdentifier($subTable),
+                BigqueryQuote::quoteSingleIdentifier($filter->getRefColumn()),
+            );
+
+            $query->andWhere(
+                sprintf(
+                    '%s %s (SELECT %s FROM %s.%s)',
+                    $column,
+                    self::OPERATOR_MULTI_VALUE[$filter->getOperator()],
+                    $subColumn,
+                    BigqueryQuote::quoteSingleIdentifier(
+                        ProtobufHelper::repeatedStringToArray($filter->getRefPath())[0],
+                    ),
+                    BigqueryQuote::quoteSingleIdentifier($subTable),
```

Comment:
tady bych iteroval nad polem path a pak k tomu table. Jde o to, ze kdybychom nahodou to delali cross-projektove a pridali do path jeste projekt, tak to zacne auto-magicky fungovat. 

takze neco jako `array_map([BigqueryQuote, 'quoteSingleIdentifier'], array_merge(ProtobufHelper::repeatedStringToArray($filter->getRefPath()), subTable))`

neber me za slovo, nevim poradi u array_map :)